src/app/shared/address/tf-address.component.scss

```diff
@@ -0,0 +1,12 @@
+@import './../sass/_index';
+
+li {
+  color:$black;
```

Comment:
Do not use `li` . Use `BEM css` model. Always try to use element's `class-name` as css selector